core/src/main/scala/Config.scala

```diff
@@ -395,7 +471,12 @@ object Config {
     val nomadcfg = readNomad(cfg.subconfig("nelson.nomad"))
 
     val gitcfg = readGithub(cfg.subconfig("nelson.github"))
-    val git = new Github.GithubHttp(gitcfg, http0)
+    val git: GithubOp ~> Task = gitcfg match {
```

Comment:
We should use something other than `GithubOp` now

Reply:
Same idea as before. I left that kind of changes for a second review.